plugins/disk/stratis

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/python3 -tt
+# -*- coding: utf-8 -*-
+
+"""Munin plugin to monitor stratis pools and filesystems.
+
+Copyright 2020 Kim B. Heino, Foobar Oy
+License GPLv2+
+
+#%# capabilities=autoconf
+#%# family=auto
+"""
+
+import os
+import subprocess
+import sys
+
+
+def safename(variable):
+    """Return safe variable name."""
+    ret = []
+    for letter in variable:
+        if letter.isalnum():
+            ret.append(letter)
+        else:
+            ret.append('_')
+    return ''.join(ret)
+
+
+def run_binary(arg):
+    """Run binary and return output."""
+    try:
+        cmd = subprocess.Popen(
+            arg, shell=False, close_fds=True, bufsize=-1,
+            stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        return cmd.communicate()[0].decode('utf-8', 'ignore')
+    except OSError:
+        return ''
+
+
+def parse_unit(number, unit):
+    """Parse "1.60 TiB" to bytes."""
+    number = float(number)
+    if unit == 'TiB':
+        return number * 1024 * 1024 * 1024 * 1024
+    if unit == 'GiB':
+        return number * 1024 * 1024 * 1024
+    if unit == 'MiB':
+        return number * 1024 * 1024
+    if unit == 'KiB':
+        return number * 1024
+    return number
+
+
+def find_pools():
+    """Return list of found pools and filesystems."""
+    pool = []
+    for line in run_binary(['/usr/bin/stratis', 'pool']).splitlines():
+        if line.startswith('Name  '):
+            continue
+        line = line.split()
+        total = parse_unit(line[1], line[2])
+        used = parse_unit(line[4], line[5])
+        free = parse_unit(line[7], line[8])
+        pool.append((line[0], total, used, free))
+
+    files = []
+    dflist = run_binary(['/usr/bin/df']).splitlines()
```

Comment:
I think, you can use `df` instead of the absolute path. This could improve portability.